2019/day2/day2.go

```diff
@@ -0,0 +1,102 @@
+package day2
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
```

Comment:
One thing I like to do is to assign types for different things such as OpCode, Address. It helps to prevent some typos.

Reply:
good idea :)